schema.sql

```diff
@@ -6,6 +6,7 @@ CREATE TABLE animals (
   date_of_birth DATE,
   escape_attempts INTEGER default 0,
   neutered BOOLEAN,
-  weight_kg DECIMAL
+  weight_kg DECIMAL,
+  species VARCHAR(100)
```

Comment:
- [ ] The way to document that we are changing the structure of the animal's table is by using the `Alter table` SQL statement. This helps users to see the development of our database. Kindly do so
e.g
`ALTER TABLE animals ADD species varchar(255);`